pkg/kubectl/resource_printer.go

```diff
@@ -716,6 +740,13 @@ func printPodList(podList *api.PodList, w io.Writer, options PrintOptions) error
 func printPodTemplate(pod *api.PodTemplate, w io.Writer, options PrintOptions) error {
 	name := pod.Name
 	namespace := pod.Namespace
+	kind := options.KindName
+
+	if options.WithKind {
+		if _, err := fmt.Fprintf(w, "%s\t", kind); err != nil {
```

Comment:
I am afraid no. But I do think we should refactor `kind&namespace` check out to avoid code duplication.